pkg/kv/kvserver/gc/gc.go

```diff
@@ -115,6 +116,20 @@ var MaxIntentKeyBytesPerCleanupBatch = settings.RegisterIntSetting(
 	settings.NonNegativeInt,
 )
 
+// AdmissionPriority determines the admission priority level to use for MVCC GC
+// work.
+var AdmissionPriority = settings.RegisterEnumSetting(
```

Comment:
nit: I dont think a setting is necessary, but also have no objections to adding one.